lib/explorer/series.ex

```diff
@@ -1819,6 +1819,22 @@ defmodule Explorer.Series do
     end
   end
 
+  @doc """
+  Returns [...] #Help!
+
+  ## Examples
+
+      iex> s1 = Explorer.Series.from_list([1, 2, 3])
+      iex> Explorer.Series.is_in(s1, [1, 2])
+      #Explorer.Series<
+        Polars[3]
+        boolean [true, true, false]
+      >
+  """
+  def is_in(left, right) do
+    Shared.apply_series_impl(:is_in, [left, right])
```

Comment:
I think you can copy the implementation from `equal`, because it is the same type of comparison?

Reply:
> I think you can copy the implementation from `equal`, because it is the same type of comparison, except this one occurs with all elements?

Can I? When I try using the same implementation from  `equal`,  the comparison with lists stop working, see below:

```Elixir
# Keeps working
      iex> s1 = Explorer.Series.from_list([1, 2, 3])
      iex> Explorer.Series.is_in(s1, [1, 2])
      #Explorer.Series<
        Polars[3]
        boolean [true, true, false]
      >

# Stops working
      iex> s1 = Explorer.Series.from_list([1, 2, 3])
      iex> Explorer.Series.is_in(s1, [1, 2])
      #Explorer.Series<
        Polars[3]
        boolean [true, true, false]
      >
```

I guess I would need to make changes on `valid_for_bool_mask_operation?/2`, right?

Reply:
> I think you can copy the implementation from `equal`, because it is the same type of comparison, except this one occurs with all elements?

Can I? When I try using the same implementation from  `equal`,  the comparison with lists stop working, see below:

```Elixir
# Keeps working
      iex> s3 = Explorer.Series.from_list([1, 2, 3])
      iex> s4 = Explorer.Series.from_list([1, 2, 4])
      iex> Explorer.Series.is_in(s3, s4)
      #Explorer.Series<
        Polars[3]
        boolean [true, true, false]
      >

# Stops working
      iex> s1 = Explorer.Series.from_list([1, 2, 3])
      iex> Explorer.Series.is_in(s1, [1, 2])
      #Explorer.Series<
        Polars[3]
        boolean [true, true, false]
      >
```

I guess I would need to make changes on `valid_for_bool_mask_operation?/2`, right?